src/sentry/notifications/notification_action/action_validation.py

```diff
@@ -2,27 +2,19 @@
 
 from django.core.exceptions import ValidationError
 
-from sentry.constants import ObjectStatus
+from sentry.integrations.discord.actions.issue_alert.form import DiscordNotifyServiceForm
 from sentry.integrations.msteams.actions.form import MsTeamsNotifyServiceForm
-from sentry.integrations.services.integration.model import RpcIntegration
-from sentry.integrations.services.integration.service import integration_service
 from sentry.integrations.slack.actions.form import SlackNotifyServiceForm
 from sentry.notifications.notification_action.registry import action_validator_registry
 from sentry.workflow_engine.models.action import Action
 
 from .types import BaseActionValidatorHandler
 
 
-def _get_integration(validated_data: dict[str, Any], provider: str) -> RpcIntegration:
+def _get_integration_id(validated_data: dict[str, Any], provider: str) -> str:
     if not (integration_id := validated_data.get("integration_id")):
         raise ValidationError(f"Integration ID is required for {provider} action")
-
-    integration = integration_service.get_integration(
-        integration_id=integration_id, status=ObjectStatus.ACTIVE
-    )
```

Comment:
Removed checking for the existence of the integration in the translation layer because this is already handled inside legacy forms. Slack example
https://github.com/getsentry/sentry/blob/b9804fff2a6edfe89bca928b937430cb90f9612c/src/sentry/integrations/slack/actions/form.py#L96-L101

Reply:
ty for this note.